pt/cheatsheet-deep-learning.md

```diff
@@ -1,291 +1,291 @@
 **1. Deep Learning cheatsheet**
 
-&#10230;
+&#10230; Resumo de Aprendizagem Profunda
 
 <br>
 
 **2. Neural Networks**
 
-&#10230;
+&#10230; Redes Neurais
 
 <br>
 
 **3. Neural networks are a class of models that are built with layers. Commonly used types of neural networks include convolutional and recurrent neural networks.**
 
-&#10230;
+&#10230; Redes neurais são uma classe de modelos que são construídos com camadas. Os tipos de redes neurais comumente utilizadas incluem redes neurais convolucionais e recorrentes.
 
 <br>
 
 **4. Architecture ― The vocabulary around neural networks architectures is described in the figure below:**
 
-&#10230;
+&#10230; Arquitetura - O vocabulário em torno das arquiteruras de redes neurais é descrito na figura abaixo:
 
 <br>
 
 **5. [Input layer, hidden layer, output layer]**
 
-&#10230;
+&#10230; [Camada de entrada, camada escondida, camada de saída]
 
 <br>
 
 **6. By noting i the ith layer of the network and j the jth hidden unit of the layer, we have:**
 
-&#10230;
+&#10230; Note que i é a i-ésima camada da rede e j a j-ésima unidade escondida da camada, nós temos:
```

Comment:
I used `Dado que` or `Definindo que` as a translation to `By noting`. Because I think this is a mathematical definition. What do you think?

Reply:
Nice, I was thinking about a better term. Those would be better. I will look for them along the file to make this substitution.